keps/sig-node/2570-memory-qos/README.md

```diff
@@ -601,6 +615,7 @@ are missing a bunch of machinery and tooling and can't do that now.
 <!--
 Even if applying deprecation policies, they may still surprise some users.
 -->
+No
```

Comment:
(comment actually for unchanged question above about rollout / upgrade)

Is there an API change associated with this? I see the feature gate is only kubelet. If there is an API change, then rollout is more complicated, as you need to deal with the gate set differently in api server and kubelet.

Reply:
There's no API change associated with this.